src/express.js

```diff
@@ -7,17 +7,23 @@ const compression = require('compression');
 const { healthcheckRoute } = require('./healthcheck.js');
 const { handleNoMatch, handleError } = require('./errors.js');
 
-function start(app, config) {
+function notifySuccess(app, callback) {
+  callback && callback(null, { app });
+}
+
+function start(app, config, callback) {
     if (String(process.env.ENABLE_CLUSTER).toLowerCase() === 'true' && cluster.isMaster) {
```

Comment:
process.env variables are always strings (if they are set), so this would work and be more flexible...
```
    if (process.env.ENABLE_CLUSTER && process.env.ENABLE_CLUSTER.search(/^(1|t)/i) !== -1 && cluster.isMaster) {
```

Reply:
I'll leave that bit alone since it isn't part of the code I changed